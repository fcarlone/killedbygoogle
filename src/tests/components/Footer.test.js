import React from 'react';
import { shallow } from 'enzyme';
import FooterContainer from '../../components/Footer';

test('should render FooterContainer correctly', () => {
  const wrapper = shallow(<FooterContainer />);
  expect(wrapper).toMatchSnapshot();
});
