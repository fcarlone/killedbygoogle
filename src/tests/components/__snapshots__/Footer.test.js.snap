// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render FooterContainer correctly 1`] = `
<styled.footer>
  <a
    href="https://twitter.com/killedbygoogle"
  >
    @killedbygoogle
  </a>
   - 
  <a
    href="https://github.com/codyogden/killedbygoogle"
  >
    GitHub
  </a>
   - 
  <a
    href="https://codyogden.com"
  >
    Cody Ogden
  </a>
</styled.footer>
`;
